lib/management/push.js

```diff
@@ -261,8 +272,30 @@ function addOverlayMessageListener(callback) {
     overlayMessageListener = callback;
 }
 
+function startPushConnectionHealthCheckServer(cb) {
+    const server = http.createServer((req, res) => {
+        const path = parse(req.url).pathname;
+
+        if (path !== '/_/healthcheck') {
+            res.writeHead(404);
+            res.write('Not Found');
+        } else if (connected) {
+            res.writeHead(200);
+            res.write('Connected');
+        } else {
+            res.writeHead(503);
+            res.write('Not Connected');
+        }
+
+        res.end();
+    });
+
+    server.listen(_config.port, cb);
```

Comment:
I am assuming "push health check" servers and cloudserver will run in different containers and so you did not have to specify new ports for them.

Reply:
Yes, it's only used in the 2 new scripts that are not using the whole s3 server/routes and have a different view of what is considered healthy.